test_runner/regress/test_proxy.py

```diff
@@ -340,3 +340,39 @@ def qq(
     assert headers["Neon-Batch-Deferrable"] == "true"
 
     assert result[0]["rows"] == [{"answer": 42}]
+
+
```

Comment:
Maybe also add a test `test_sql_over_http_pool_wrong_password`? To test the returned error when password is wrong.

Reply:
I have a feeling that the test database ignores passwords entirely, as I cannot get it to fail (even when the logs indicate that it's connecting to postgres with the password directly)

Reply:
> `local   all             all                                     trust`

looks like it

Reply:
Managed to modify it to work, hopefully it doesn't break every single other test 😂